### PR TITLE
Information api call

### PR DESCRIPTION
**Pull request by Morel-Victor**

### CODE REVIEW

File: src/Components/AccordionUser/Information/index.jsx [hunk 1]
```diff
@@ -1,30 +1,112 @@
...
+    lastname,
+    firstname,
+    phone,
+    streetNumber,
+    street,
+    city,
+    postalCode,
+  });
+  const dispatch = useDispatch();
+  // const { id } = useParams();
```
Comment (Caroline-CD) [inline src/Components/AccordionUser/Information/index.jsx:28]: Can you delete the comment please ?

**Review by Caroline-CD — CHANGES_REQUESTED** [review]
Can you delete the comment please ?

**Review by Caroline-CD — DISMISSED** [review]

File: src/Components/AccordionUser/Connexion/style.jsx [hunk 2]
```diff
@@ -1,13 +1,108 @@
...
+  padding: 5px 0 5px 10px;
+  animation-duration: 0.55s;
+  animation-fill-mode: forwards;
+  animation-name: ${(props) => (props.toggle ? openAnimation : closeAnimation)};
+  transition: 0.5s linear all;
+  @media (max-width: 1400px) {
+    max-width: 80vw;
+    height: 330px;
+    display: flex;
+    /* flex-direction: column; */
```
Comment (Hacene010) [inline src/Components/AccordionUser/Connexion/style.jsx:38]: dead code !

File: src/Components/AccordionUser/Information/index.jsx [hunk 3]
```diff
@@ -1,30 +1,111 @@
...
+      .then((response) => {
+        const user = response.data;
+        dispatch({ type: 'LOGGEDIN', user });
+        toast('Vos informations ont changer');
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+  console.log(informations);
```
Comment (Hacene010) [inline src/Components/AccordionUser/Information/index.jsx]: no console.log

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Seen a small evils

**Review by Caroline-CD — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**